lib/redcloth.rb

```diff
@@ -5,14 +5,10 @@
 # appears to be fixed in Edge Rails [51e4106].
 Object.send(:remove_const, :RedCloth) if Object.const_defined?(:RedCloth) && RedCloth.is_a?(Class)
 
-require 'rbconfig'
 begin
-  conf = Object.const_get(defined?(RbConfig) ? :RbConfig : :Config)::CONFIG
-  prefix = conf['arch'] =~ /mswin|mingw/ ? "#{conf['MAJOR']}.#{conf['MINOR']}/" : ''
-  lib = "#{prefix}redcloth_scan"
-  require lib
+  require 'redcloth_scan'
```

Comment:
@MSP-Greg : can you please check the changes proposed in #62 ?

There is rescue that only changes the value of lib if the rescue is fired.
That seems a safer approach while I am still situating myself.

Reply:
> That seems a safer approach while I am still situating myself.

PR #62 is fine.

If there's no expectation that pre-compiled gems be released, this cleans up the code accordingly.

Reply:
Right now I am trying to clean the PR and issues, but if you think this will help you I will be happy to spend some time.
But if it helps you to release windows pre-compiled version I will be happy to spend some time on it.

I never used windows in my 20+ years as a software developer and this is the first time I took over maintenance of a gem, so I am still trying to situate myself.

Reply:
> I never used windows

No problem, pretty common with Ruby users.

I've used it since DOS days.  Windows & macOS desktops, run Ubuntu 22.04 & 20.04 on Windows WSL2.

Re this PR, it can be closed, as #62 (merged) fixes the issue.  It just leaves some code that isn't applicable to current releases (just a standard gem)...

Reply:
I am remembering now that I actually used windows in the past few years, both Ruby and some powershell... but I am happy to not have to do that anymore.

My first programming language was C for SunOS... And I worked with C and Sparc Solaris for 1/2 of my programming career...

Anyway, I am interested in getting RedCloth with a good windows support because you use it and you've been a key contributor for several years! 💯